app/views/casework/crime_applications/_income.html.erb

```diff
@@ -12,29 +12,35 @@
         <%= t(income_details.income_above_threshold, scope: 'values') %>
       </dd>
     </div>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:has_frozen_income_or_assets) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= t(income_details.has_frozen_income_or_assets, scope: 'values') %>
-      </dd>
-    </div>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:client_owns_property) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= t(income_details.client_owns_property, scope: 'values') %>
-      </dd>
-    </div>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:has_savings) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= t(income_details.has_savings, scope: 'values') %>
-      </dd>
-    </div>
+    <% unless income_details.has_frozen_income_or_assets.nil? %>
```

Comment:
Before and after of fix 
<img width="814" alt="Screenshot 2024-01-04 at 14 43 26" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/51047911/1d279e30-9fa9-49ef-844a-45c26df5110b">